pages/categories/operational.md

```diff
@@ -78,6 +78,12 @@ Software-as-a-Service (SaaS) applications.
 {% include linkedHeading.html heading="Managed Services" level=3 %}
 Managed Security Operations, IT support and data backup services.
 
+{% include linkedHeading.html heading="Offsite Employee Devices" level=3 %}
+
+The emissions incurred by computers (desktops, laptops), monitors, printers, and other devices used by employees, when used out-of-office. Since these are required by the business, they are accounted for, but as with other indirect groupings, the business does not control them directly.
```

Comment:
Maybe mention something about them technically being under a different scope from the point of view of GHG Protocol reporting?

Reply:
Isn't that implied by the groupings (Cat O/Cat C labels with scope numbers)?